styles/light/_light-theme.scss

```diff
@@ -672,7 +668,7 @@ pre {
       ##{$company} {
         .biz-logo {
           max-height: 40px;
-          content: url("../img/uses/lightmode/#{$company}.#{$type}");
+          content: url("../img/uses/lightmode/#{$company}.#{$type}"); 
```

Comment:
Why are you adding whitespace to the end of these lines? That seems unnecessary and clutters the diff.